pom.xml

```diff
@@ -135,10 +135,24 @@
             <version>2.0.16</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- Spring integration (optional!) -->
         <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
-            <version>2.0.16</version>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
It seems like liquibase does not follow the convention, but it would be great to have `spring-boot-liquibase-opensearch-starter` / `liquibase-opensearch-starter`  as a separate module